packages/webapp/src/inductions/components/induction-journeys/inviter-witnesses/video-form.tsx

```diff
@@ -39,13 +39,13 @@ export const InductionVideoForm = ({
         setIsLoading(false);
     };
 
-    const getVideoUrl = () => {
+    const videoUrl = useMemo(() => {
```

Comment:
👏

Reply:
Yeah. it was still re-rendering when you hit the submit button. This fixed that.